packages/ckeditor5-table/src/tableproperties/tablepropertiesediting.ts

```diff
@@ -255,14 +267,17 @@ function enableAlignmentProperty( schema: Schema, conversion: Conversion, defaul
 		// Support for the `align` attribute as the backward compatibility while pasting from other sources.
 		.attributeToAttribute( {
 			view: {
+				name: 'table',
```

Comment:
This was provided on the model side, but that notation was not supported, and as a result, this converter was triggered on any element with the given attribute.